lib/govuk_app_config/govuk_healthcheck.rb

```diff
@@ -0,0 +1,14 @@
+require "govuk_app_config/govuk_healthcheck/checkup"
+require "govuk_app_config/govuk_healthcheck/active_record" if defined?(ActiveRecord)
+require "govuk_app_config/govuk_healthcheck/sidekiq_redis"
```

Comment:
Does this also need to be guarded with `if defined?(Sidekiq)`?

Reply:
I'm not certain either are needed tbh, I ran info-frontend with this code and it doesn't have either sidekiq or AR

Reply:
Removed from both.